plugins/GranularPitchShifter/GranularPitchShifterControls.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * GranularPitchShifterControls.h
+ *
+ * Copyright (c) 2024 Lost Robot <r94231/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef GRANULARPITCHSHIFTER_CONTROLS_H
+#define GRANULARPITCHSHIFTER_CONTROLS_H
+
+#include "EffectControls.h"
+#include "GranularPitchShifterControlDialog.h"
+
+constexpr float GPS_RANGE_SECONDS[5] = {5, 10, 40, 40, 120};
```

Comment:
As previously mentioned, this isn't the naming convention we use for global constants. This can also be moved into an anonymous namespace in`GranularPitchShifterControls.cpp` I believe.